lib/config/config.js

```diff
@@ -260,24 +268,23 @@ function setSelfSettings (app) {
 }
 
 function readSettingsFile (app) {
-  const settings = getSettingsFilename(app)
-  if (!app.argv.s && !fs.existsSync(settings)) {
-    console.log('Settings file does not exist, using empty settings')
-    app.config.settings = {}
-  } else {
-    debug('Using settings file: ' + settings)
-    app.config.settings = require(settings)
-  }
-  if (_.isUndefined(app.config.settings.pipedProviders)) {
-    app.config.settings.pipedProviders = []
```

Comment:
Are you sure this is unneeded? And interfaces below? Could very well be.

Reply:
The diff is showing strangely. The logic is there still, I it just got moved down a few lines.

Reply:
I'm returning early if `app.config.disableWriteSettings` is true.